tox.ini

```diff
@@ -7,7 +7,9 @@ deps =
     python-subunit
     junitxml
     coverage
-    coverage-enable-subprocess
+
+    # this is pinned since it's a bit of an obscure package.
```

Comment:
the general idea here is to avoid a situation where somebody releases a compromised `coverage-enable-subprocess 1.0.1`, and since we're the only people on the internet who use it, we don't notice.

Reply:
Looks like we're not pinning it in the `py35-old` testenv (line 91)... but I also question how useful it is to pin just *this* specific dependency.

Reply:
fair, I guess. for now I've added the pin to line 91. I could remove it altogether if you prefer.

Reply:
Eh, might as well leave it and we can fight about it another day 😉